subsys/bluetooth/mesh/proxy_srv.c

```diff
@@ -69,16 +58,9 @@
 	.interval_min = BT_GAP_ADV_FAST_INT_MIN_2,                             \
 	.interval_max = BT_GAP_ADV_FAST_INT_MAX_2
```

Comment:
These `ADV_` defines could be moved to proxy.h, I believe. They're duplicated in proxy_srv.c and pb_gatt.c, but in pb_gatt.c, they're handling `CONFIG_BT_MESH_PROXY_USE_DEVICE_NAME` wrong